packages/reactive-element/package.json

```diff
@@ -381,7 +383,8 @@
     "/node/"
   ],
   "dependencies": {
-    "@lit-labs/ssr-dom-shim": "^1.1.2"
+    "@lit-labs/ssr-dom-shim": "^1.1.2",
+    "lit-html": "^3.1.0"
```

Comment:
The build step is likely failing because of this `lit-html` dependency being added without a `wireit` build dependency.

So a clean build of `reactive-element` won't currently build `lit-html` as a dependency. This then shows up as `Cannot find module 'lit-html/static.js' or its corresponding type declarations.`.

This is also a race condition, because a top level `npm run build`, will build everything, and in the case where `lit-html` is built before `reactive-element`, everything will be fine. In the error case, a re-run of `npm run build` will work because `lit-html` will have been built.

The fix is to add the wireit dependency `"../lit-html:build:ts:types"` to the wireit `"build:ts"` command. On L188.

Reply:
Ah I see, the wireit cache is used in the build pipe. Thanks!

Reply:
Thanks!